scripts/lit.js

```diff
@@ -6,20 +6,39 @@
 // Configuration:
 //
 // Commands:
-// randbot lit <message_link> - Immortalizes a message in #lit... fam
+//   hubot lit <message_link> - Immortalizes a message in #lit... fam
+//   hubot lit - Immortalizes a shared message in #lit... fam
 //
 // Author:
 //   mikestephens/harasho
 
+const { failureMessages, successMessages } = require('./utils/lit.messages');
+
+// TODO: make this a configurable values
+// paro: Just for the time being, let's keep these separate.
+const channel = 'lit';
+const validURL = 'https://hashtaggaming.slack.com/archives/';
+
+function hasAttachments(rawMessage) {
+  return rawMessage !== undefined &&
+    rawMessage.attachments !== undefined &&
+    rawMessage.attachments.length > 0;
+}
+
 module.exports = (robot) => {
-  robot.respond(/lit (.*)/i, (res) => {
-    // TODO: make this a configurable value
-    if (res.match[1].includes('https://hashtaggaming.slack.com/archives/')) {
-      res.send('I gotchu fam');
-      robot.messageRoom('lit', res.match[1]);
+  robot.respond(/lit\s?(.*)?/i, (res) => {
+    let param = res.match[1];
+
+    if (hasAttachments(res.message.rawMessage)) {
+      param = res.message.rawMessage.attachments[0].from_url;
+    }
+
+    if (param !== undefined && param.includes(validURL)) {
+      res.send(successMessages.random());
+      robot.messageRoom(channel, param);
     }
     else {
-      res.send("The fuck you talking 'bout fam?");
+      res.send(failureMessages.random());
```

Comment:
Same thing with the failure messages here, I'd prefer using `res.send(res.random())`